src/mmw/js/src/analyze/views.js

```diff
@@ -253,15 +257,22 @@ var TabContentView = Marionette.LayoutView.extend({
             })
         }));
 
-        this.tableRegion.show(new TableView({
-            units: units,
-            collection: census
-        }));
+        if (this.model.get('name') !== 'livestock') {
```

Comment:
Consider reversing this `if` block so there is a more direct relationship between the condition and the body:

```javascript
if (this.model.get('name') === 'livestock') {
  // new LivestockTableView
} else {
  // new ChartView
  // new TableView
}
```